packet/flatcar-linux/kubernetes/workers/cl/install.yaml.tmpl

```diff
@@ -88,14 +88,63 @@ storage:
             mkfs.ext4 /dev/md/node-local-storage
           }
 
+          function create_disk_specific_data_raid() {
```

Comment:
Doesn't this function superseed `create_data_raid()`? Can't we use just one?

Reply:
For backwards compatibility I created this new function and did not touch the old functionality. The old thing works as it was.

Reply:
I strongly think we shouldn't duplicate code because we are afraid to change it. Like, this is c&p and within the same file, just to not break something.

And the functionality is quite superseeded... let's use the new one, and try to make sure that it works for the old case too.

What do you think?

Reply:
There are lot of changes if you look closely, like the device name is different. Also to get around the problem of longer code is to host the script somewhere and then pull the script with the functionality that ignition already provides.

Reply:
(Just a comment on the Ignition size: I haven't found a limit for Packet and guess that it will be larger than AWS, so I wouldn't worry now.)